examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Ironnode Examples
+
+This directory contains some examples of using the IronNode SDK to protect sensitive data.
+
+- dek-reuse shows how to reduce the number of calls to ironnode, and corresponding interactions
+with IronCore's service, but reusing a key for multiple data items
+
+
+To run an example, just change to the directory and do a `yarn start`.
```

Comment:
This line should also be in `dek-reuse/README.md` in case they skip this upper level readme.